impl.hpp

```diff
@@ -852,7 +876,7 @@ bool Client::run_download() noexcept {
     }
     auto now = std::chrono::steady_clock::now();
     std::chrono::duration<double> elapsed = now - begin;
-    client_side_speed = compute_speed_kbits(  //
+    this->download_speed_ = compute_speed_kbits(  //
```

Comment:
Needless '//'

Reply:
I think it was here to prevent some auto-formatting tool from changing this line. AFAIK @bassosimone does not use any such tool, I don't either, so I've removed it.